tools/ds-identify

```diff
@@ -747,11 +759,15 @@ probe_floppy() {
     [ -b "$fpath" ] ||
         { STATE_FLOPPY_PROBED=1; return 1; }
```

Comment:
(I didn't realise we had this. I also recently learned we're removing floppy support from FreeBSD in 15 or 16)

Reply:
That floppy check is only for RHEV (though I though some versions of VMware did/do make use of floppy also).